### ISSUE TITLE
Garbled Graphics on Samsung devices - v.0.6.12

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#319]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker

Alpha version of v0.6.12 available:
http://andorstrail.com/viewtopic.php?f=6&t=2082

What steps will reproduce the problem?

1.Installed v.0.6.12 over v.0.6.11

2.Spiratic jumbled maps issues on new and old save game files.

3.After uninstalling AT, then reinstalling AT, deleting all AT data still left 
on device, maps work good with new save games.

4.Then, reloading previous saves will reproduce graphical errors, on old and 
new save slots.

What is the expected output? What do you see instead?

Need to see normal maps instead of garbled ones.

What version of the product are you using? On what device?

Sarumar: Samsung Galaxy 3 / Android OS - 4.0.4
AlphaOmega: Samsung Galaxy S Blaze / Android OS - Gingerbread

Please provide any additional information below.

Normal troubleshooting this has failed with uninstalling and re-installing.  I 
believe that users will be unable to continue to use their save games from 
v.0.6.11 in the new v.0.6.12.

I have tried to repeat the effect on my tablet - Coby Kyros 9742 rooted / 
Android ICS 4.0.3.  However have been unable to reproduce the effect.  So far 
it is limited to these 2 users running on different Samsung Galaxy devices.

```

Original issue reported on code.google.com by `Tale...@gmail.com` on 17 Sep 2012 at 6:47

**Comment by GoogleCodeExporter** [comment]
```
Add user & device:
Hardy: HTC Sensation z710e / Android OS - 4.1.1 (MIUI build 2.9.14)
```

Original comment by `Tale...@gmail.com` on 17 Sep 2012 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
Not solely a Samsung issue.  The following user is unable to reproduce the 
results on his device:

Tek:  Samsung Galaxy S3 / Android OS - 4.0.4
```

Original comment by `Tale...@gmail.com` on 17 Sep 2012 at 11:21

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report.
Fixed in 0814c3f42ee519e3d6f72cecfe8267afb07a3e9b.
```

Original comment by `oskar.wi...@gmail.com` on 22 Oct 2012 at 6:15
- Changed state: **Ready**
- Added labels: **Milestone-0.6.12**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:44
- Changed state: **Fixed**
